spacy/visualization.py

```diff
@@ -0,0 +1,601 @@
+from os import linesep
+from typing import Dict, List, Optional, Union, cast
```

Comment:
```suggestion
from typing import Dict, List, Optional, Union, cast
from os import linesep
```
This is a convention we should probably document, but the `from typing` imports are typically the very first line.